src/components/ToggleSmall/ToggleSmall.js

```diff
@@ -0,0 +1,78 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+const ToggleSmall = ({
+  className,
+  defaultToggled,
+  toggled,
+  onChange,
+  onToggle,
+  id,
+  ariaLabel,
+  ...other
+}) => {
+  let input;
+  const wrapperClasses = classNames({
+    'bx--form-item': true,
+    [className]: className,
+  });
+
+  const checkedProps = {};
+
+  if (typeof toggled !== 'undefined') {
+    checkedProps.checked = toggled;
+  } else {
+    checkedProps.defaultChecked = defaultToggled;
+  }
+
+  return (
+    <div className={wrapperClasses}>
+      <input
+        {...other}
+        {...checkedProps}
+        type="checkbox"
+        id={id}
+        className="bx--toggle bx--toggle--small"
+        onChange={evt => {
+          onChange && onChange(evt);
+          onToggle(input.checked, id, evt);
+        }}
+        ref={el => {
+          input = el;
+        }}
+        aria-label={ariaLabel}
+      />
+
+      <label className="bx--toggle__label" htmlFor={id}>
+        <span className="bx--toggle__appearance">
+          <svg
+            className="bx--toggle__check"
+            width="6px"
+            height="5px"
+            viewBox="0 0 6 5">
+            <path d="M2.2403 2.7299L4.9245 0 6 1.1117 2.2384 5 0 2.6863 1.0612 1.511z" />
+          </svg>
+        </span>
+      </label>
+    </div>
+  );
+};
+
+ToggleSmall.propTypes = {
+  className: PropTypes.string,
+  defaultToggled: PropTypes.bool,
+  onToggle: PropTypes.func,
+  id: PropTypes.string.isRequired,
+  toggled: PropTypes.bool,
+  small: PropTypes.bool,
```

Comment:
Did you mean to remove this prop (along with the `defaultProps`)?

Reply:
Oops, I don't need the small prop. Is that what you are referring to, or something else?

Reply:
Yes @alisonjoseph it’s what exactly I meant. Thank you for your quick fix!